src/tools/imgtool/iflopimg.cpp

```diff
@@ -137,28 +137,28 @@ static imgtoolerr_t imgtool_floppy_open_internal(imgtool::image *image, imgtool_
 
 
 
-static imgtoolerr_t imgtool_floppy_open(imgtool::image *image, imgtool_stream *f)
+static imgtoolerr_t imgtool_floppy_open(imgtool::image *image, imgtool::stream *f)
 {
 	return imgtool_floppy_open_internal(image, f, FALSE);
 }
 
 
 
-static imgtoolerr_t imgtool_floppy_create(imgtool::image *image, imgtool_stream *f, util::option_resolution *opts)
+static imgtoolerr_t imgtool_floppy_create(imgtool::image *image, imgtool::stream *f, util::option_resolution *opts)
 {
 	floperr_t ferr;
 	imgtoolerr_t err = IMGTOOLERR_SUCCESS;
 	struct imgtool_floppy_image *fimg;
 	const imgtool_class *imgclass;
 	const struct FloppyFormat *format;
-	imgtoolerr_t (*create)(imgtool::image *, imgtool_stream *, util::option_resolution *);
-	imgtoolerr_t (*open)(imgtool::image *image, imgtool_stream *f);
+	imgtoolerr_t (*create)(imgtool::image *, imgtool::stream *, util::option_resolution *);
+	imgtoolerr_t (*open)(imgtool::image *image, imgtool::stream *f);
```

Comment:
No, there isn't - I agree that it is desirable to do so, my concern is about "doubling down" on the intrusiveness of this particular change.  This is the same reason that I transitioned imgtool::image and imgtool::partition separately.

In addition to better adoption of references, there other things I'd like to do in follow up patches - change to std::function, change char buffers to std::string, make functions return unique_ptr etc.